test-harness/src/snapshots/toolchain__literals into-coq.snap

```diff
@@ -47,13 +47,13 @@ Export Hax_lib (t_int).
 
 Record t_Foo : Type :=
   {
-    f_field : t_u8;
+    t_Foo_f_field : t_u8;
```

Comment:
This is a solution to overlapping field names in structs.

Reply:
Oh, I see!
Then you need to improve the name policies API: the binding `enum_constructor_name_transform` is what you want, but you need to add an extra argument "struct_name" or something, just as I did for `enum_constructor_name_transform`.
Do you see what I mean?

Reply:
Ahh, yes. Makes sense.

Reply:
This will also fix the issue I had, with overwriting keywords (e.g. `end`).

Reply:
Great! :tada: